tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1146,6 +1150,7 @@ qos/test_qos_sai.py::TestQosSai::testQosSaiHeadroomPoolWatermark:
   xfail:
     reason: "Headroom pool size not supported."
     conditions:
+      - "hwsku in ['Force10-S6100'] and topo_type in ['t1-64-lag']"
```

Comment:
@XuChen-MSFT for those 2 conditions under testQosSaiHeadroomPoolWatermark case, the relationship between them is  AND or OR, in my understanding, it's OR, right? Miss the following line?

conditions_logical_operator: or

Reply:
filed github issue #12292